content/docs/fundamentals/converting/resolving-errors@id.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: Mengatasi error validasi
+$order: 2
+toc: true
+---
+
+[TOC]
+
+Pada bagian ini, kita akan membahas dan menyelesaikan error validasi AMP dari halaman AMP.  Perhatikan bahwa error mungkin muncul dalam urutan yang berbeda di konsol Anda.
+
+## Sertakan charset
+
+Kita akan memulai dengan memperbaiki error berikut:
+
+<pre class="error-text">
+Tag wajib 'meta charset=utf-8' tidak ada atau salah.
+</pre>
+
+Untuk menampilkan teks dengan benar, AMP mewajibkan Anda menentukan charset untuk halaman. Informasi charset meta juga harus menjadi turunan pertama tag `<head>`. Tag ini harus menjadi turunan pertama untuk menghindari konten yang ditafsirkan ulang ditambahkan sebelum tag charset meta.
+
+*Tambahkan** kode berikut sebagai baris pertama tag `<head>`:
+
+
+```html
+<meta charset="utf-8" />
+```
+
+**Simpan** file dan muat ulang halaman. Verifikasi bahwa error charset sudah tidak muncul.
+
+## Sertakan link kanonis
+
+Sekarang, mari kita lihat error berikut:
+
+<pre class="error-text">
+Tag wajib 'link rel=canonical' tidak ada atau salah.
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).

Reply:
Updated code block with EN version.